pkgs/test_analyzer_plugin/README.md

```diff
@@ -0,0 +1,13 @@
+# test_analyzer_plugin
```

Comment:
We should probably add something to the package:test readme as well, informing users of this plugin and how to enable it for their project.

Reply:
Since we are very "soft launching" analyzer plugins ATM, I think we don't want to advertise it there yet. WDYT?

Reply:
Up to you I guess, but we won't get usage if we don't advertise it